### PR TITLE
invoke tests directly with pytest

### LINKED ISSUE #353: most of the tests are failing on master now
```
$ tox
GLOB sdist-make: /home/tt/g/user-cont/ogr/setup.py 
py36 inst-nodeps: /home/tt/g/user-cont/ogr/.tox/.tmp/package/1/ogr-0.11.1.dev4+g022e9df.zip
WARNING: Discarding $PYTHONPATH from environment, to override specify PYTHONPATH in 'passenv' in your configuration.
py36 installed: amqp==2.5.0,-e git+git@github.com:TomasTomecek/ansible-bender.git@e3ef97fb7cdb4e19bb8f21266c09d6fdc5e76392#egg=ansible_bender,aspy.yaml==1.2.0,attrs==19.3.0,
beanbag==1.9.2,billiard==3.6.0.0,celery==4.3.0,certifi==2019.11.28,cffi==1.14.0,cfgv==1.5.0,chardet==3.0.4,click==7.1.1,colin-redhat==0.0.2,conu==0.6.0,coverage==5.0.3,crypt
ography==2.8,Deprecated==1.2.5,dnspython==1.16.0,dpath==1.4.2,eventlet==0.25.0,flexmock==0.10.4,frambo==0.0.3,generator==0.1.dev34+g6448aff,gevent==1.4.0,gitchangelog==3.0.4
,gitdb==4.0.2,GitPython==3.1.0,identify==1.4.0,idna==2.9,importlib-metadata==0.8,jsl==0.2.4,kobo==0.7.0,koji==1.16.0,kombu==4.6.3,kubernetes==8.0.2,monotonic==1.5,more-itert
ools==8.2.0,nodeenv==1.3.3,numpy==1.17.2,-e git+git@github.com:TomasTomecek/ogr@022e9dff7fd02e2f4d34ca836ac298127d7f5694#egg=ogr,packaging==20.3,-e git+git@github.com:TomasT
omecek/packit.git@25a1ca9a2d746b493ee10aced1ce477cfd529653#egg=packitos,persistentdict==0.1.0,pluggy==0.13.1,pre-commit==1.14.4,py==1.8.1,pycparser==2.20,PyGithub==1.46,PyJW
T==1.7.1,pyparsing==2.4.6,pystache==0.5.4,pytest==5.4.1,pytest-cov==2.8.1,python-gitlab==2.1.2,PyYAML==5.3,raven==6.10.0,redis==3.2.1,requests==2.23.0,-e git+git@github.com:
TomasTomecek/requre.git@f5410b0095d31e2b0f27b92d7ca24e365b472d33#egg=requre,rlsbot-test==1.0.0,rpm-py-installer==0.8.0,-e git+git@github.com:TomasTomecek/generator.git@65a43
07bcd0d8f45b3c7ed8b757be5d44601a62d#egg=sandcastle,sentry-sdk==0.12.2,show-me==0.1.dev25+g72bc7ba,six==1.14.0,smmap==3.0.1,tool==0.1.0.dev0,ucho==0.1.0,-e git+git@github.com
:TomasTomecek/upsint.git@972ba36e63c8b44eedf291ac1d30f87954258ef6#egg=upsint,urllib3==1.25.8,vine==1.3.0,wcwidth==0.1.8,wrapt==1.11.1,zipp==0.3.3
py36 run-test-pre: PYTHONHASHSEED='2399286198'
py36 run-test: commands[0] | pytest --color=yes --verbose --showlocals --cov=ogr --cov-report=term-missing
============================================================================ test session starts ============================================================================
platform linux -- Python 3.6.10, pytest-5.4.1, py-1.8.1, pluggy-0.13.1 -- /home/tt/g/user-cont/ogr/.tox/py36/bin/python
cachedir: .tox/py36/.pytest_cache
rootdir: /home/tt/g/user-cont/ogr
plugins: cov-2.8.1
collected 255 items                                                                                                                                                         

tests/integration/test_factory.py::FactoryTests::test_get_project_github FAILED                                                                                       [  0%]
tests/integration/test_factory.py::FactoryTests::test_get_project_gitlab FAILED                                                                                       [  0%]
tests/integration/test_factory.py::FactoryTests::test_get_project_pagure FAILED                                                                                       [  1%]
tests/integration/test_github.py::Comments::test_issue_comments FAILED                                                                                                [  1%]
tests/integration/test_github.py::Comments::test_issue_comments_author FAILED                                                                                         [  1%]
tests/integration/test_github.py::Comments::test_issue_comments_author_regex FAILED                                                                                   [  2%]
tests/integration/test_github.py::Comments::test_issue_comments_regex FAILED                                                                                          [  2%]
tests/integration/test_github.py::Comments::test_issue_comments_regex_reversed FAILED                                                                                 [  3%]
tests/integration/test_github.py::Comments::test_issue_comments_reversed FAILED                                                                                       [  3%]
tests/integration/test_github.py::Comments::test_issue_comments_updates FAILED                                                                                        [  3%]
tests/integration/test_github.py::Comments::test_pr_comments FAILED                                                                                                   [  4%]
tests/integration/test_github.py::Comments::test_pr_comments_author FAILED                                                                                            [  4%]
tests/integration/test_github.py::Comments::test_pr_comments_author_regex FAILED                                                                                      [  5%]
```

```
E                   requre.exceptions.ItemNotInStorage: Keys not in storage:/home/tt/g/user-cont/ogr/tests/integration/test_data/test_github/tests.integration.test_github.Ge
nericCommands.test_description.yaml Matched: ['SKIP tests.integration.test_github', 'SKIP ogr.services.github.project', 'SKIP github.MainClass', 'SKIP github.Requester', 'SK
IP requests.sessions', 'SKIP requre.objects'], Missing: ['requests.sessions', 'send']
```

Happens even when I use older pygithub.

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes #353

Also fixes some test failures, let's see...

### CODE REVIEW

**Comment by TomasTomecek** [comment]
let's see now

gotta say that fc31's python-gitlab is pretty ancient: the errors were so bizzare

File: ogr/services/github/issue.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def get_list(
         if assignee:
             parameters["assignee"] = assignee
         if labels:
-            parameters["labels"] = labels
+            parameters["labels"] = [project.github_repo.get_label(la) for la in labels]
```
Comment (lachmanfrantisek) [inline ogr/services/github/issue.py]: The name `la` makes no sense to me.

Comment (TomasTomecek) [reply ogr/services/github/issue.py]: changed to label

Comment (lachmanfrantisek) [reply ogr/services/github/issue.py]: Thanks, sorry for nitpicking...

**Pull request merged**